### PR TITLE
Entry Season Request

### PR DESCRIPTION
**Pull request by Desnoo**
Functionality to gain info about the seasons in which the entry was made.

### CODE REVIEW

File: library/src/main/java/com/proxerme/library/connection/info/entity/Season.java [hunk 1]
```diff
@@ -0,0 +1,145 @@
...
+    String type;
+    @Body(name = "year")
+    int year;
+    @Body(name = "season")
+    int season;
+
+    /**
+     * The private constructor.
+     */
+    @SuppressWarnings("unused")
```
Comment (rubengees) [inline library/src/main/java/com/proxerme/library/connection/info/entity/Season.java]: Are you sure we should start doing this? That would be maaaaany methods to annotate :D

Comment (Desnoo) [inline library/src/main/java/com/proxerme/library/connection/info/entity/Season.java]: :D I dont believe that anyone has interest in this xD I will change it.

**Comment by rubengees** [comment]
The build failed due to incorrectly placed annotations. Could you have a look? (Details -> Artifacts -> lint-library-results.html)

**Comment by Desnoo** [comment]
Fixed. Was a stupid error^^

**Comment by rubengees** [comment]
Okay, looks good now. I'll merge when the build succeedes. Thanks again for the PR :+1:

**Comment by Desnoo** [comment]
Alright. :+1:

**Pull request merged**